ecosystem/sep-0030.md

```diff
@@ -257,7 +258,10 @@ Name | Type | Description
 `identities` | array | A list of identities that if any one is authenticated will be able to gain control of this account.
 `identities[].role` | string | Same as request. See above.
 `identities[].authenticated` | boolean | Optional. Present and true if the currently authenticated client is authenticated with the identity.
-`signer` | string | The signer public key that the server will sign transactions with for this account when requested by an authenticated user.
+`signer` | string | The most recently added signer public key that the server will sign transactions with for this account when requested by an authenticated user.
+`signers` | array | The signers' public keys that the server can sign transactions with for this account when requested by an authenticated user, ordered from most recently added to least recently added.
+`signers[].signer` | array | The signer public key that the server can sign transactions with for this account when requested by an authenticated user.
+`signers[].generated_at` | array | The ISO8601 timestamp of when the signer was generated and became available for signing.
```

Comment:
Why do we choose to use ISO8601 over RFC3339? Maybe using RFC3339 is sufficient given the complete set of format in ISO8601 is quite large and more complex?

Reply:
That's a good point. I think many programming languages reference ISO8601 and so we might be making it easier to for folks to mention it but also it is essentially a subset so it is probably fine. I'm primarily thinking of Java that has ISO8601 parsers and generators and not RFC3339. I don't feel strongly and in practice I don't think it will matter. Anyone else have any thoughts on this?

Reply:
I'm going to mention both instead since mostly RFC3339 is a profile of ISO8601. Ultimately it will have little impact though.
```suggestion
`signers[].generated_at` | array | The RFC3339/ISO8601 timestamp of when the signer was generated and became available for signing.
```